ScoreSaber/UI/Elements/Profile/ProfileDetailView.cs

```diff
@@ -227,6 +435,26 @@ private void SetCrowns(string playerId) {
             }
         }
 
+        private void SetRichStatus(Scene scene) {
+            switch (scene) {
+                case Scene.offline:
+                    profilePrefixPicture = "ScoreSaber.Resources.Offline.png";
+                    break;
+                case Scene.online:
+                    profilePrefixPicture = "ScoreSaber.Resources.Online.png";
+                    break;
+                case Scene.menu:
+                    profilePrefixPicture = "ScoreSaber.Resources.Online.png";
+                    break;
+                case Scene.playing:
+                    profilePrefixPicture = "ScoreSaber.Resources.Online.png";
+                    break;
```

Comment:
should use fallthrough cases to avoid the code (and in particular constant) duplication and make it easier to read.

Reply:
Fixed

Reply:
Not quite what I meant, but better than before. Still would prefer the fallthrough cases though.

Reply:
fixed

Reply:
I think Qwasyx meant to use case grouping, looks like this:

```cs
private void SetRichStatus(Scene scene) {
    const string online = "ScoreSaber.Resources.Online.png";
    const string offline = "ScoreSaber.Resources.Offline.png";

    switch (scene) {
        case Scene.offline:
            profilePrefixPicture = offline;
            break;
        case Scene.menu:
        case Scene.playing:
        case Scene.online:
            profilePrefixPicture = online;
            break;
        default:
            profilePrefixPicture = offline;
            break;
    }
}
```